changelogs/current.yaml

```diff
@@ -56,6 +56,10 @@ bug_fixes:
   change: |
     Fixes a bug where the lifetime of the HttpConnectionManager's ActiveStream can be out of sync
     with the lifetime of the codec stream.
+- area: rbac filter
```

Comment:
If it's a regression introduced by a PR, we don't need a record for it in the changelog